scripts/xtensa-build-zephyr.sh

```diff
@@ -140,10 +145,12 @@ build()
 			       -l "$STAGING"/sof/sof-"$platform".ldc \
 			       "$bdir"/zephyr/zephyr.elf
 
+                        test "$RIMAGE_KEY" = 'None' ||
 			west sign  --build-dir "$bdir" \
```

Comment:
Indentation looks odd here. Same with line below.

Reply:
Tabs vs spaces, the new line seems to be indented with spaces.